gensrc/thrift/PaloInternalService.thrift

```diff
@@ -136,6 +136,8 @@ struct TQueryOptions {
   // see BE config `max_pushdown_conditions_per_column` for details
   // if set, this will overwrite the BE config.
   30: optional i32 max_pushdown_conditions_per_column
+  // whether enable spilling to disk
+  31: optional bool enable_spilling = false;
```

Comment:
Please add this variable to doc.

Reply:
The variable already in the doc。
![image](https://user-images.githubusercontent.com/10553413/84489076-b7cd8d80-acd3-11ea-963c-0f66b87f224e.png)

I will change some detail of the doc.

Reply:
OK